File: pset6/mario/mario.py
from cs50 import get_int
height = get_int("Height:")
while height <= 0 or height > 8:
    height = get_int("Height:")
if height == 1:
    print("#  #")
elif height == 2:
    print(" #  #")
    print("##  ##")
elif height == 3:
    print("  #  #")
    print(" ##  ##")
    print("###  ###")
elif height == 4:
    print("   #  #")
    print("  ##  ##")
    print(" ###  ###")
    print("####  ####")
elif height == 5:
    print("    #  #")
    print("   ##  ##")
    print("  ###  ###")
    print(" ####  ####")
    print("#####  #####")
elif height == 6:
    print("     #  #")
    print("    ##  ##")
    print("   ###  ###")
    print("  ####  ####")
    print(" #####  #####")
    print("######  ######")
elif height == 7:
    print("      #  #")
    print("     ##  ##")
    print("    ###  ###")
    print("   ####  ####")
    print("  #####  #####")
    print(" ######  ######")
    print("#######  #######")
elif height == 8:
    print("       #  #")
    print("      ##  ##")
    print("     ###  ###")
    print("    ####  ####")
    print("   #####  #####")
    print("  ######  ######")
    print(" #######  #######")
    print("########  ########")

File: pset1/mario/mario.c
#include <cs50.h>
#include <stdio.h>

int main(void)
{
    int height = get_int("Height:\n");

    while (height <= 0 || height > 8)
    {
        height = get_int("Height:\n");
    }


     if (height == 1)
         {
            printf("#  #\n");
        }
        else if (height == 2)
        {
            printf(" #  #\n");
            printf("##  ##\n");
        }
        else if (height == 3)
        {
            printf("  #  #\n");
            printf(" ##  ##\n");
            printf("###  ###\n");
        }
        else if (height == 4)
        {
            printf("   #  #\n");
            printf("  ##  ##\n");
            printf(" ###  ###\n");
            printf("####  ####\n");
        }
        else if (height == 5)
        {
            printf("    #  #\n");
            printf("   ##  ##\n");
            printf("  ###  ###\n");
            printf(" ####  ####\n");
            printf("#####  #####\n");
        }
        else if (height == 6)
        {
            printf("     #  #\n");
            printf("    ##  ##\n");
            printf("   ###  ###\n");
            printf("  ####  ####\n");
            printf(" #####  #####\n");
            printf("######  ######\n");
        }
        else if (height == 7)
        {
            printf("      #  #\n");
            printf("     ##  ##\n");
            printf("    ###  ###\n");
            printf("   ####  ####\n");
            printf("  #####  #####\n");
            printf(" ######  ######\n");
            printf("#######  #######\n");
        }
        else if (height == 8)
          {
            printf("       #  #\n");
            printf("      ##  ##\n");
            printf("     ###  ###\n");
            printf("    ####  ####\n");
            printf("   #####  #####\n");
            printf("  ######  ######\n");
            printf(" #######  #######\n");
            printf("########  ########\n");
          }
}